tableau/v_tableau.surveys.kipp_forward_survey.sql

```diff
@@ -0,0 +1,80 @@
+USE gabby
+GO
+
+--CREATE OR ALTER VIEW tableau.kipp_forward_survey
```

Comment:
uncomment

Reply:
Hey, this was an old/unfinished file, deleting in the next commit.